src/main/java/com/startsteps/ecommerceapi/service/commands/builder/OrderBuilder.java

```diff
@@ -32,7 +32,7 @@ public OrderBuilder shoppingCart(ShoppingCart shoppingCart){
         this.shoppingCart = shoppingCart;
         return this;
     }
-    public OrderBuilder orderService(OrderService orderService) {
```

Comment:
it would be better if this stays as an interface

Reply:
thank you, I'll change it if I have time because it requires to refactor the code a bit.